.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+### Connected issues (if any)
+<!--- link to the issues here or leave blank -->
+
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the **CONTRIBUTING** document.
+- [ ] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
+- [ ] My code follows the code style of this project.
```

Comment:
Currently, we do not have any codystyle guide besides mypy and flake tests. Let's remove this option for now not to confuse contributors.